docs/en/operation/Metrics-System.md

```diff
@@ -126,3 +126,87 @@ to the under store.
 
 Tags are additional pieces of metadata for the metric such as user name or under storage location.
 Tags can be used to further filter or aggregate on various characteristics.
+
+## Setting Grafana for Alluxio
+
+Grafana is a metics analytics and visualization software commonly used for visualizing time series
+data. You can use Grafana to better visualize the various metrics that Alluxio collects. The software
+allows users to more easily see what's going on behind the scenes in Alluxio.
+
+Although before setting up Grafana a monitoring tool must be set up that Grafana will pull metrics from
+such as `Prometheus` or `Graphite`.
```

Comment:
I would add here that specifically Alluxio supports exporting metrics Prometheus and Graphite.

I also recommend not starting this sentence with the word "Although"